src/Microsoft.AspNetCore.Razor.Language/BoundAttributeDescriptorComparer.cs

```diff
@@ -76,15 +75,7 @@ public virtual int GetHashCode(BoundAttributeDescriptor descriptor)
 
             var hashCodeCombiner = HashCodeCombiner.Start();
             hashCodeCombiner.Add(descriptor.Kind);
-            hashCodeCombiner.Add(descriptor.IsIndexerStringProperty);
-            hashCodeCombiner.Add(descriptor.IsEnum);
-            hashCodeCombiner.Add(descriptor.HasIndexer);
             hashCodeCombiner.Add(descriptor.Name, _stringComparer);
-            hashCodeCombiner.Add(descriptor.IndexerNamePrefix, _stringComparer);
-            hashCodeCombiner.Add(descriptor.TypeName, StringComparer.Ordinal);
-            hashCodeCombiner.Add(descriptor.IndexerTypeName, StringComparer.Ordinal);
-            hashCodeCombiner.Add(descriptor.Documentation, StringComparer.Ordinal);
-            hashCodeCombiner.Add(descriptor.DisplayName, StringComparer.Ordinal);
```

Comment:
Some comparers still add DisplayName to hashCode. Should be consistent one way or the other.

Reply:
Sure, I'll remove it from more places